### PR TITLE
Add look and feel page

### PR DESCRIPTION
**Pull request by dcwalk**
Closes https://github.com/hyphacoop/organizing/issues/77

This needs refinement, but an initial pass.

### CODE REVIEW

File: look-and-feel.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# Look and Feel
+
+### Name and wordmark
```
Comment (benhylau) [inline look-and-feel.md:3]: How come we skipped a level?

Comment (dcwalk) [reply look-and-feel.md:3]: format to size

File: look-and-feel.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Our name uses title case with the following variations:
+
+> **HYPHA**  
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)).
```
Comment (garrying) [inline look-and-feel.md:11]: `data-proofer-ignore` attr could go on the LinkedIn anchor to get around the proofer error.

File: look-and-feel.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+
+### Preferred icons and emoji
+
+We tend to use the following emoji and iconography: 
+
+🍄 `:mushroom:`  
+⚑  
+
+### Assets and Templates
```
Comment (garrying) [inline look-and-feel.md:60]:
```suggestion
### Assets and templates
```

File: look-and-feel.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+Body typeface 
+>
+
+Body typeface (web)
+> `font-family: -apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial, sans-serif`
+
+### Colours
+
+<img src="images/hypha-colours.png" alt="Hypha colours" width="90%"/>
```
Comment (garrying) [inline look-and-feel.md:30]: Could the text on the Lime swatch be translucent black?

File: look-and-feel.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+Registration Black
```
Comment (garrying) [inline look-and-feel.md:32]: I associate "Registration Black" to the CMYK process black. I think X11 is just "Black"

File: look-and-feel.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+`#00ff00;`  
+`rgb(0, 255, 0);`  
+
+
+### Preferred icons and emoji
+
+We tend to use the following emoji and iconography: 
+
+🍄 `:mushroom:`  
+⚑
```
Comment (garrying) [inline look-and-feel.md:58]:
```suggestion
⚑  `&#9873;`
```

Comment (garrying) [reply look-and-feel.md:58]: Maybe include the 🏴 emoji as well?

File: look-and-feel.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+🍄 `:mushroom:`  
+⚑  
+
+### Assets and Templates
+
+- Wordmark
+- Image bank
+- Letterhead 
+- Report Template
```
Comment (garrying) [inline look-and-feel.md:65]:
```suggestion
- Report template
```

File: look-and-feel.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+> **HYPHA** Worker Co-operative  
+> Hypha Worker Co-operative Inc. [our legal name]  
+
+We use the following wordmark in online spaces, services and social media where we represent the organization (e.g., [Twitter](https://twitter.com/hyphacoop) and [LinkedIn](https://ca.linkedin.com/company/hyphacoop)). 
+
+<img src="images/hypha-wordmark.png" alt="Hypha Wordmark in white on purple background" width="50%"/>
+
+### Typography
+
+Our **heading typeface** is [Work Sans](https://fonts.google.com/specimen/Work+Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```
Comment (garrying) [inline look-and-feel.md:17]: Proxima Nova could be listed as a secondary typeface for now if we go by the deck

Comment (garrying) [inline look-and-feel.md:17]:
```suggestion
Our **heading typeface** is [Work Sans](https://github.com/weiweihuanghuang/Work-Sans) and our **secondary (body) typeface** is !!!!. For the web we prefer using the [system font stack](https://css-tricks.com/snippets/css/system-font-stack/) sans-serif typeface.
```

File: look-and-feel.md (hunk 5 above)
Comment (dcwalk) [reply look-and-feel.md:32]: Okay, was just using the names that I was told were associated!

File: look-and-feel.md (hunk 6 above)
Comment (dcwalk) [reply look-and-feel.md:58]: Sure, if we want to be anarchists about it ;)

File: look-and-feel.md (hunk 2 above)
Comment (dcwalk) [reply look-and-feel.md:11]: gitbook uses a custom parser based on kramdown from what I can tell (most docs disappearing), so I can't just pass this with the {: } syntax. I'm resolving this problem by moving where the content is (not essential here anyway). :wink:

File: look-and-feel.md (hunk 8 above)
Comment (dcwalk) [reply look-and-feel.md]: Just realized this isn't a free font :flushed:

**Comment by dcwalk** [comment]
Okay, addressed feedback! Stripped out the extra bits on typefaces because I'm running up on limitations on adding fonts and having them viewable in gitbook. Chalk another thing up to using a discontinued tool for this :grimacing:

**Comment by dcwalk** [comment]
@garrying if I could get another quick review I think maybe we could merge in the first iteration for now?

**Pull request merged**